### PR TITLE
Add icons for Asus Router integration

### PR DESCRIPTION
**Pull request by Vaskivskyi**
## Proposed change

This PR adds images for the proposed new integration Asus Router (`asus_router`). Since all the images should be the same as for the already existing `asuswrt`, I suppose the correct way is the link

## Type of change

- [x] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [ ] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: https://github.com/home-assistant/core/pull/86084
- Link to documentation pull request: https://github.com/home-assistant/home-assistant.io/pull/25805
- Link to integration documentation on our website: 

## Checklist

- [ ] The added/replaced image(s) are **PNG**
- [ ] Icon image size is 256x256px (`icon.png`)
- [ ] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [ ] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [ ] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

File: core_integrations/asus_router [hunk 1]
```diff
@@ -0,0 +1 @@
+asuswrt
```
Comment (frenck) [inline core_integrations/asus_router:1]: These are not files,  ou need to create a symbolic link.

Comment (Vaskivskyi) [reply core_integrations/asus_router:1]: Sorry. Seems like I did not think a lot. I hope, now this is fine

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**